src/04_pinning/01_chapter.md

```diff
@@ -314,7 +314,7 @@ Let's see how pinning and the `Pin` type can help us solve this problem.
 
 The `Pin` type wraps pointer types, guaranteeing that the values behind the
 pointer won't be moved. For example, `Pin<&mut T>`, `Pin<&T>`,
-`Pin<Box<T>>` all guarantee that `T` won't be moved even if `T: !Unpin`.
+`Pin<Box<T>>` all guarantee that `T` won't be moved if `T: !Unpin`.
```

Comment:
I think this is difficult to understand because the above sentence ("The `Pin` type wraps...") gives an explanation that can misunderstand like "the pointed value will not move unconditionally".

```diff
  The `Pin` type wraps pointer types, guaranteeing that the values behind the
- pointer won't be moved. For example, `Pin<&mut T>`, `Pin<&T>`,
+ pointer won't be moved if it is not implementing `Unpin`. For example, `Pin<&mut T>`, `Pin<&T>`,
  `Pin<Box<T>>` all guarantee that `T` won't be moved if `T: !Unpin`.
```